server/routerlicious/packages/test-utils/src/testDocumentStorage.ts

```diff
@@ -121,6 +121,7 @@ export class TestDocumentStorage implements IDocumentStorage {
             },
             sequenceNumber,
             lastClientSummaryHead: undefined,
+            lastSummarySequenceNumber: -1,
```

Comment:
We always start from sequence number 1. So the default value should be 0 everywhere.